frontend/src/modules/alarms/AlarmsPage.tsx

```diff
@@ -292,41 +294,71 @@ export const AlarmsPage = (): JSX.Element => {
     setPage(value);
   };
 
-  const alarmLimitsRequestStandby = useSelector(getAlarmLimitsRequestStandby);
+  const alarmLimitsRequestDraft = useSelector(getAlarmLimitsRequestDraft);
+  const alarmLimitsCurrent = useSelector(getAlarmLimitsCurrent);
   const dispatch = useDispatch();
   const currentMode = useSelector(getParametersRequestMode);
   const ventilating = useSelector(getParametersIsVentilating);
-  const setAlarmLimits = (data: Partial<AlarmLimitsRequest>) => {
+  const alarmLimitsUnsaved = useSelector(getAlarmLimitsUnsavedChanges);
+  const [alarmLimits] = useState((alarmLimitsCurrent as unknown) as Record<string, Range>);
```

Comment:
Can we get rid of this `useState` and directly use the selector (typecasting it into a `Record<string, Range>` if needed)?